script/run.sh

```diff
@@ -43,5 +43,11 @@ if [ -n "$HOME" ]; then
     VOLUMES="$VOLUMES -v $HOME:$HOME"
 fi
 
+# Only allocate tty if we detect one
+if [ -t 1 ]; then
+    DOCKER_RUN_OPTIONS="-ti"
+else
+    DOCKER_RUN_OPTIONS="-i"
```

Comment:
But if you're doing `docker-compose run -d` (detached) that won't be present.  I guess my question, is can I run: `ssh -C "docker-compose run -d"` without the `-t` ssh flag.   Will docker-compose blow up because there is no input?